rush-plugins/rush-audit-cache-plugin/src/core/Analyzer.ts

```diff
@@ -84,26 +71,52 @@ export class AuditCacheAnalyzer {
         if (!project) {
           throw new Error(`no project ${projectName} RushConfigurationProject`);
         }
+        const readFileResolver: ReadFileResolver = new ReadFileResolver();
+        const writeFileResolver: WriteFileResolver = new WriteFileResolver();
+
         const { projectFolder } = project;
 
+        // get user project rush audit cache config
+        const rushProjectAuditCacheJsonPath: string = path.join(
+          projectFolder,
+          RUSH_AUDIT_CACHE_JSON_RELATIVE_PATH
+        );
+
+        const rushProjectAuditCacheJson: IAuditCacheFileFilter | undefined =
+          tryLoadJson<IAuditCacheFileFilter>(rushProjectAuditCacheJsonPath);
+
+        const { fileFilters = [] } = rushProjectAuditCacheJson ?? {
+          fileFilters: [],
+        };
+
+        // prepare project safe read and write matcher
         const projectSafeReadMatcher: Ignore = ignore();
+        const projectSafeWriteMatcher: Ignore = ignore();
```

Comment:
Is it possible to encapsulate the safeXXMatcher into the corresponding Resolver class?